src/index.html

```diff
@@ -6,7 +6,69 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <ul class="page__calendar calendar calendar--month-length-29 calendar--start-day-mon">
+      <li class="calendar__day calendar__day--1"></li>
+
+      <li class="calendar__day calendar__day--2"></li>
+
+      <li class="calendar__day calendar__day--3"></li>
```

Comment:
Here empty lines between similar list elements don't increase readability, but the size of the html.

Reply:
@Mykola-Maik for sure